src/Resources/views/tailwind_2/_table.html.twig

```diff
@@ -255,6 +271,7 @@
                     class="whatwedo_table-subtable hidden"
                     {{ stimulus_target('araise/table-bundle/accordion', 'content') }}
                 >
+                    <th class="w-12 px-3 py-3"></th>
```

Comment:
It seems like this might be wrong:

- I don't think the `th` should appear in the `tr` with `td`s in the `<tbody>`. It makes no sense semantically. This should be a `td` at best.
- There shouldn't be an empty table cell just for lay outing purpose, as this will be an empty cell in the accessibility tree too, which is confusing. Should this be column spanned with its previous cell then?